README.md

```diff
@@ -87,39 +87,25 @@ A minimal example of using an existing configuration.nix:
 
   outputs = { self, darwin, nixpkgs }: {
     darwinConfigurations."Johns-MacBook" = darwin.lib.darwinSystem {
-      system = "x86_64-darwin";
       modules = [ ./configuration.nix ];
     };
   };
 }
 ```
 
-Inputs from the flake can also be passed to `darwinSystem`. These inputs are then
+Inputs from the flake can also be passed into `darwinSystem`. These inputs are then
 accessible as an argument `inputs`, similar to `pkgs` and `lib`, inside the configuration.
 
 ```nix
 darwin.lib.darwinSystem {
-  system = "x86_64-darwin";
   modules = [ ./configuration.nix ];
-  inputs = { inherit darwin dotfiles nixpkgs; };
+  specialArgs = { inherit inputs; };
 }
```

Comment:
```suggestion
{
  outputs = inputs@{ self, darwin, nixpkgs }: {
    darwinConfigurations."Johns-MacBook" = darwin.lib.darwinSystem {
      modules = [ ./configuration.nix ];
      specialArgs = { inherit inputs; };
    };
  };
}
```

I think this will help show new users how to define `inputs`

Reply:
I added `inputs@` to the flake in the previous code block and edited the `configuration.nix` one to reflect the inputs that are in that one. I think that's probably good enough, since it's immediately after the example flake that sets the context for the suggestion. (Otherwise the entire example would have to be duplicated: `darwin` is (frustratingly) not the name in the global Nix registry, and I'm a bit leery about suggesting people use `nixpkgs` with no branch reference.)